src/views/mfa-verify/PushForm.js

```diff
@@ -76,16 +76,22 @@ export default Form.extend({
   },
   setSubmitState: function (ableToSubmit) {
     const button = this.$el.find('.button');
-
+    const a11ySpan = this.$el.find('.accessibility-text');
     this.enabled = ableToSubmit;
     if (ableToSubmit) {
       button.removeClass('link-button-disabled');
       button.prop('value', loc('oktaverify.send', 'login'));
       button.prop('disabled', false);
+      if (a11ySpan) {
+        a11ySpan.remove();
+      }
     } else {
       button.addClass('link-button-disabled');
       button.prop('value', loc('oktaverify.sent', 'login'));
       button.prop('disabled', true);
+      this.add(
+        `<span class='accessibility-text' role='alert'>${loc('oktaverify.sent', 'login')}</span>`,
```

Comment:
Using `role="alert"` isn't ideal, but since the user interacts with this button with the intent to perform an action - it seemed appropriate.

@edburyenegren-okta - Think this is OK?

Reply:
I think so! The push response is time-sensitive, so it's best if they hear the trigger asap. Great work here, @jmelberg-okta